### PR TITLE
Getting_In_Touch: Add information about long issue discussions

### PR DESCRIPTION
**Pull request by ladaradon**
Closes https://github.com/coala-analyzer/documentation/issues/45

### CODE REVIEW

File: Help/Getting_In_Touch.rst [hunk 1]
```diff
@@ -11,7 +11,8 @@ following things:
 -  Subscribe to our `mailing lists <https://github.com/coala-analyzer/coala/wiki/Mailing-Lists>`__.
 -  Open an issue. Whether you want a new feature for coala or you have found
    a bug, just file `an issue <https://github.com/coala-analyzer/coala/issues>`__
-   and we will check it out!
+   and we will check it out. In case of long discussion, you should create
+   new issue with clearly defined task!
```
Comment (sils1297) [inline Help/Getting_In_Touch.rst]: shouldn't it be "a new issue"?

**Comment by sils1297** [comment]
Hi,

it seems that your commit message doesn't contain enough information. Don't worry, most people don't know about good commit messages when they join us, including very experienced developers :) Standards vary from project to project.

The most important part is the first line of your commit message - maintainers will use it to read through a large number of commits and thus it's important that it's short but most importantly concise. I'd heartily recommend reading http://coala.readthedocs.io/en/latest/Getting_Involved/Writing_Good_Commits.html !

To change your message you just use `git commit --amend` and then `git push --force` the new commit to replace the old one.

Cheers!

**Comment by sils1297** [comment]
Hey, I think you'll want to use Closes instead of Fixes - I'll talk about that in a few minutes in the workshop :)

**Comment by sils1297** [comment]
you'll want to reference the issue, not the PR!

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/documentation/pull/88/commits/8a97fbdfbc97902ac57b5f43d7e5b5b3531d9ba9

**Comment by sils1297** [comment]
@rultor merge

**Comment by Asnelchristian** [comment]
@rultor merge

**Comment by Asnelchristian** [comment]
check https://github.com/coala-analyzer/documentation/pull/91 :)

**Pull request closed without merging**